### DISCUSSION TITLE
Stability SD-XL ControlNet LORAs are out

### DISCUSSION THREAD

**Discussion by Alchete** [vladmandic/sdnext#2034] [General]
https://huggingface.co/stabilityai/control-lora
https://www.reddit.com/r/StableDiffusion/comments/15uwomn/stability_releases_controlloras_efficient/

🙏

**Comment by midcoastal** [comment]
Saw this (a while ago) and have them.  Wondering what the blocker is on enabling ControlNet for SD-XL in SD.Next is.  @vladmandic ?
